### PR TITLE
Limit code consolidation

### PR DESCRIPTION
**Pull request by AndreasAlbert**

### CODE REVIEW

**Review by siqiyyyy — APPROVED** [review]
Looks good to me!

**Review by alpakpinar — COMMENTED** [review]
Looks good, thanks!
Only thing that got my attention is that in limit.py, we might want to load mjj if vbf channel is selected. I don't think we're loading it in the legacy_vbf file.

File: bucoffea/limit/limit.py [hunk 1]
```diff
@@ -5,24 +5,42 @@
 from bucoffea.plot.util import acc_from_dir
 from bucoffea.limit.legacy import legacy_limit_input
```
Comment (alpakpinar) [inline bucoffea/limit/limit.py]: Just an additional quick comment: Should we change this legacy file that we are importing from? I think the relevant file is renamed now.

**Comment by AndreasAlbert** [comment]
> Looks good, thanks!
> Only thing that got my attention is that in limit.py, we might want to load mjj if vbf channel is selected. I don't think we're loading it in the legacy_vbf file.

Good point, done

File: bucoffea/limit/limit.py (hunk 1 above)
Comment (AndreasAlbert) [reply bucoffea/limit/limit.py]: Yes, this import it broken. Thanks for catching it!
I'm surprised flake8 didnt catch that...

**Pull request merged**